### DISCUSSION TITLE
I am facing issue while cloning the repo:

### DISCUSSION THREAD

**Discussion by Kushaldotel** [TareqMonwer/Django-School-Management#163] [General]
PS E:\School-Management\B> git clone https://github.com/TareqMonwer/Django-School-Management.git
Cloning into 'Django-School-Management'...
remote: Enumerating objects: 9578, done.
remote: Counting objects: 100% (2270/2270), done.
remote: Compressing objects: 100% (1894/1894), done.
error: RPC failed; curl 92 HTTP/2 stream 5 was not closed cleanly: CANCEL (err 8)
error: 3871 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output
PS E:\School-Management\B> git clone --verbose https://github.com/TareqMonwer/Django-School-Management.git
>>
Cloning into 'Django-School-Management'...
POST git-upload-pack (185 bytes)
POST git-upload-pack (812 bytes)
remote: Enumerating objects: 9578, done.
remote: Counting objects: 100% (2270/2270), done.
remote: Compressing objects: 100% (1894/1894), done.
error: RPC failed; curl 92 HTTP/2 stream 5 was not closed cleanly: CANCEL (err 8)
error: 5652 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output
PS E:\School-Management\B> git clone --depth 1 https://github.com/TareqMonwer/Django-School-Management.git
>> 
Cloning into 'Django-School-Management'...
remote: Enumerating objects: 4408, done.
remote: Counting objects: 100% (4408/4408), done.
remote: Compressing objects: 100% (3922/3922), done.
error: RPC failed; curl 92 HTTP/2 stream 5 was not closed cleanly: CANCEL (err 8)
error: 2608 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output

My network is fine and GitHub is also responding. But getting issues during cloning:
PS E:\School-Management\B> ping github.com
>> 

Pinging github.com [20.205.243.166] with 32 bytes of data:
Reply from 20.205.243.166: bytes=32 time=80ms TTL=117
Reply from 20.205.243.166: bytes=32 time=81ms TTL=117
Reply from 20.205.243.166: bytes=32 time=80ms TTL=117
Reply from 20.205.243.166: bytes=32 time=83ms TTL=117

Ping statistics for 20.205.243.166:
    Packets: Sent = 4, Received = 4, Lost = 0 (0% loss),
Approximate round trip times in milli-seconds:
    Minimum = 80ms, Maximum = 83ms, Average = 81ms
PS E:\School-Management\B>

**Comment by tanvirstreame** [comment]
Can you change your network, like switching to a mobile network or something, or can you try out these solutions -
https://stackoverflow.com/questions/59282476/error-rpc-failed-curl-92-http-2-stream-0-was-not-closed-cleanly-protocol-erro
